lib/vf_common.cpp

```diff
@@ -101,6 +102,9 @@ namespace ValueFlow
         if (value_size == 0)
             return value;
 
+        // sizeof(long long) = 8
+        value_size = std::min(sizeof(MathLib::bigint), value_size);
```

Comment:
I have the feeling we want to be able to truncate according to sizeof(int) here. I.e. the result of  `~0U + 2U`  should be 1.